tools/js-sdk-release-tools/src/common/utils.ts

```diff
@@ -143,16 +145,21 @@ export function fixChangelogFormat(content: string) {
     return content;
 }
 
-export function tryReadNpmPackageChangelog(changelogPath: string): string {
+export function tryReadNpmPackageChangelog(changelogPathFromNpm: string, lastStableNpmViewParameter?: lastStableNpmViewParameter): string {
     try {
-        if (!fs.existsSync(changelogPath)) {
-            logger.warn(`NPM package's changelog '${changelogPath}' does not exist.`);
-            return "";
+        if (!fs.existsSync(changelogPathFromNpm)) {
+            logger.warn(`NPM package's changelog '${changelogPathFromNpm}' does not exist.`);
```

Comment:
```suggestion
            logger.warn(`Failed to find NPM package's changelog '${changelogPathFromNpm}'`);
```
Please make warning log and error log start with "Failed to" if possible